ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -570,6 +587,18 @@ keys:
                   type: bool
             session_tracker:
               type: str
+            shared_secret:
+              type: dict
+              keys:
+                profile:
+                  type: str
```

Comment:
make profile and hash_algorithm required keys

Reply:
Fixed.